R/field.R

```diff
@@ -37,6 +37,14 @@ merge.roxy_field <- function(x, y, ...) {
   roxy_field_simple(x$field, c(x$values, y$values))
 }
 
+#' @export
```

Comment:
I think you forgot to re-document to update the NAMESPACE?

Reply:
Done now.